src/components/asset-list/RecyclerAssetList2/core/RowRenderer.tsx

```diff
@@ -106,10 +107,17 @@ function rowRenderer(type: CellType, { uid }: { uid: string }) {
             );
           }
           case CellType.NFT: {
-            const { index, uniqueId } = data as NFTExtraData;
+            const {
+              index,
+              uniqueId,
+              onPressUniqueToken,
+            } = data as NFTExtraData & {
+              onPressUniqueToken: (asset: UniqueAsset) => void;
```

Comment:
is there any reason why the `NFTExtraData` type itself is not updated to include the `onPressUniqueToken`?

Reply:
Yeah probably not, i'll change 👍